newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -219,9 +219,25 @@ export const HomePage = React.memo<Props>(
           project,
           onSave,
           canSave,
+          showUserChip,
         ]
       );
 
+      // Ensure the toolbar is up to date when the active tab changes.
+      React.useEffect(
+        () => {
+          updateToolbar();
+        },
+        [updateToolbar]
+      );
+
+      React.useEffect(() => {
+        // Let the GetStarted page handle if the user chip should be shown.
```

Comment:
A more accurate comment could be `Always show the user chip, apart on the GetStarted page which handles it on its own`